### PR TITLE
updated train.py

### PR DESCRIPTION
**Pull request by jinxianghooi**
train.py step 4 pronunciation dictionary and dataset linking was outdated and doesn't work. Updated with reference to endpoints code.

### CODE REVIEW

File: examples/cli/elan/train.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
     ds = elpis.new_dataset('dsy')
     ds.add_directory('/recordings/transcribed', extensions=['eaf', 'wav'])
     ds.auto_select_importer() # Selects Elan because of eaf file.
-    ds.importer.set_setting('tier_name', 'Phrase')
+    ds.importer.set_setting('tier_name', 'tx')
```
Comment (nicklambourne) [inline examples/cli/elan/train.py:22]: This change is not necessary. Our canonical tests use the Abui toy corpora which uses 'Phrase as the tier name'

**Review by nicklambourne — CHANGES_REQUESTED** [review]
Can you provide output of this working on the Abui toy corpora?

File: examples/cli/elan/train.py [hunk 2]
```diff
@@ -48,7 +48,8 @@
 # Link dataset and pd to a new model, then train the model.
 if 'mx' not in elpis.list_models():
     m = elpis.new_model('mx')
-    m.link(ds, pd)
+    m.link_dataset(ds)
+    m.link_pron_dict(pd)
```
Comment (benfoley) [inline examples/cli/elan/train.py:52]: This is fine. Please reset the `tx` change above and I'm happy for this to be merged - it copies the behaviour of the model new endpoint.

**Comment by benfoley** [comment]
> Can you provide output of this working on the Abui toy corpora?
Jin sent this via Slack
[train.log](https://github.com/CoEDL/elpis/files/5182770/train.log)

**Pull request merged**